pkg/metrics/metrics.go

```diff
@@ -95,6 +89,11 @@ var (
 		"error",
 		"exporter",
 	)
+	// MapSize = defineMetric(
+	// 	"ebpf_map_size",
+	// 	"size of the eBPF maps",
+	// 	TypeGauge,
+	// )
```

Comment:
I tried to create a gauge to track the hashmap size but I didn't find a function to get the used size in bpflib...

Reply:
we can do dummy iterate like the one we use today and inside increment the map size ?

Reply:
i'd prefer to avoid any additional processing .. no worries I'll delete this commented code

Reply:
me2 :)